kotlin/workflow-tracing/src/test/resources/com/squareup/workflow/diagnostic/tracing/expected_trace_file.txt

```diff
@@ -702,6 +702,8 @@
 {"name":"used/free memory","ph":"C","ts":0,"pid":0,"tid":0,"args":{"usedMemory":1,"freeMemory":42}},
 {"name":"Snapshot","ph":"B","ts":0,"pid":0,"tid":0,"args":{}},
 {"name":"Snapshot","ph":"E","ts":0,"pid":0,"tid":0,"args":{}},
+{"name":"Worker output: TracingDiagnosticListenerTest$TestWorkflow (0)","cat":"update","ph":"i","ts":0,"pid":0,"tid":0,"s":"t","args":{"workerId":"2","description":"TypedWorker(kotlin.String)","output":"fired!"}},
+{"name":"Worker: 2","cat":"workflow","ph":"e","ts":0,"pid":0,"tid":0,"id":"workflow"},
```

Comment:
Not actually sure why this changed, but since none of the other tests broke, it's likely not a contract violation so I'm not going to spend a lot of time investigating.